### ISSUE TITLE
[Add] LightGBM (microsoft/LightGBM)

### ISSUE THREAD

**Issue by 9bow** [PyTorchKR/oss-landscape#135]
### GitHub 저장소 URL

https://github.com/microsoft/LightGBM

### 분류 (모듈)

training-inference (모델 개발/학습 및 추론)

### 카테고리

deep-learning-framework (딥러닝 프레임워크)

### 설명 (선택)

결정 트리 기반의 빠르고 분산 가능한 고성능 그래디언트 부스팅 프레임워크로, 랭킹·분류·회귀 등 다양한 ML 작업에서 정형 데이터에 대한 높은 성능과 효율을 제공합니다.

### 체크리스트

- [X] AI와 직접 관련된 오픈소스 프로젝트입니다
- [X] GitHub 저장소가 공개(public) 상태입니다
- [X] 위에서 선택한 분류가 적절합니다

---

## 추가 컨텍스트 (검토용)

- **Stars**: 18416
- **Forks**: 4021
- **License**: MIT
- **Language**: C++
- **Topics**: gradient-boosting, gbdt, gbm, decision-trees, machine-learning, distributed, kaggle
- **Homepage**: https://lightgbm.readthedocs.io/en/latest/
- **Last Pushed**: 2026-06-02
- **koreanSupport**: false (공식 한국어 문서 없음)
- **제안 태그**: gradient-boosting, machine-learning, gbdt, decision-trees

### 분류 근거

정형 데이터 ML 모델 학습의 핵심 그래디언트 부스팅 프레임워크로 training-inference 모듈에 속합니다. 고전 ML(부스팅) 전용 카테고리가 없어 프레임워크 성격상 deep-learning-framework가 가장 근접한 분류입니다(XGBoost 이슈와 동일 맥락).
